src/Stryker.CLI/Stryker.CLI/StrykerNugetFeedInfo.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Net.Http;
+
+namespace Stryker.CLI.NuGet
+{
+    public class StrykerNugetFeedInfo
+    {
+        [JsonProperty("items")]
+        private List<NuGetLibraryItem> _items = null;
+
+        public string LatestVersion
+        {
+            get
+            {
+                return _items.First().Upper;
+            }
+        }
+
+        public static StrykerNugetFeedInfo Create()
+        {
+            using (var httpclient = new HttpClient())
+            {
+                try
+                {
+                    var json = httpclient.GetStringAsync("https://api.nuget.org/v3/registration3/dotnet-stryker/index.json").Result;
```

Comment:
This is a blocking call. What if somebody runs stryker with no internet connection? Stryker should run just as expected and not crash because this check fails. Can this be made an asynchonous task?

Reply:
Almost all our processes are synchronous, there's is nothing that would run while waiting on this callback so it would be blocking anyway. If it crashes it simply returns null. Perhaps the timeout could be shorter. Or am I missing something?

Reply:
I just checked with no internet connection available the call immediately causes an exception which causes the version check to be skipped.